test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests_native.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+
+<!--
+  Copyright (c) 2019, 2019 IBM Corp. and others
+
+  This program and the accompanying materials are made available under
+  the terms of the Eclipse Public License 2.0 which accompanies this
+  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+  or the Apache License, Version 2.0 which accompanies this distribution and
+  is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+  This Source Code may also be made available under the following
+  Secondary Licenses when the conditions for such availability set
+  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+  General Public License, version 2 with the GNU Classpath
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+<!DOCTYPE suite SYSTEM "cmdlinetester.dtd">
+
+<suite id="J9 Command-Line Option Tests" timeout="600">
+    <variable name="EMPTY_OPTION" value="-Custom:EmptyOption"/>
+    <variable name="WHITESPACE_OPTION" value="-Custom:WhitespaceOption"/>
+    <variable name="MULTIPLE_WHITESPACE_OPTION" value="-Custom:MultipleWhitespaceOption"/>
+    <variable name="IGNORE_UNRECOGNIZED_OPTIONS" value="-Custom:IgnoreUnrecognizedOptions"/>
+    <variable name="COMMANDLINE_OPTION_UNRECOGNIZED" value="Command-line option unrecognised"/>
+    <variable name="TEST_NATIVEVMARGS_CONCLUDED_WITH_CODE" value="[MSG] Test nativevmargs concluded with code:"/>
+
+    <test id="test nativevmargs ignore whitespace VM option">
+        <command>$NATIVEVMARGS$ $JVMLIBPATH$ $IGNORE_UNRECOGNIZED_OPTIONS$ $WHITESPACE_OPTION$</command>
+        <output type="success" caseSensitive="yes" regex="no">$TEST_NATIVEVMARGS_CONCLUDED_WITH_CODE$ 0</output>
+        <output type="failure" regex="no">[ERR]</output>
+    </test>
+
+    <test id="test nativevmargs ignore empty VM option">
+        <command>$NATIVEVMARGS$ $JVMLIBPATH$ $IGNORE_UNRECOGNIZED_OPTIONS$ $EMPTY_OPTION$</command>
```

Comment:
Can we combine the EMPTY_OPTION test with MULTIPLE_WHITESPACE_OPTION, for the passing case, to reduce the testing cost?

Reply:
For sure! I will remove the EMPTY_OPTION test since the MULTIPLE_WHITESPACE_OPTION test should cover the passing case.

Reply:
Not exactly what I meant. The passing case can test with both options `$EMPTY_OPTION$` and `$MULTIPLE_WHITESPACE_OPTION$` on the command line.

Reply:
Ah I see, I'll make that change

Reply:
I think the tests may need to be separate, since it looks like `checkArgsConsumed()` will return as soon as the first unrecognized option is encountered: https://github.com/eclipse/openj9/blob/567bdff454ec19d0bf78c4272921b27d9d5ffdab/runtime/vm/jvminit.c#L2386-L2449

Reply:
That's why we can't combine the case where `COMMANDLINE_OPTION_UNRECOGNIZED` is expected, but we can combine the (passing) case where the arguments are expected to be ignored as if either the empty option or the whitespace option isn't ignored this test will fail. I didn't explain what I meant by the "passing case" very well, by which I meant the test with `IGNORE_UNRECOGNIZED_OPTIONS` when `$TEST_NATIVEVMARGS_CONCLUDED_WITH_CODE$ 0` is expected.

Reply:
I think I'm on the same page now 🙂 There is now one combined passing test case for empty and whitespace, and two separate tests for empty and whitespace for when `COMMANDLINE_OPTION_UNRECOGNIZED` is expected.